test.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable no-unused-vars */
+function myFunction() {
+  // make mobile nav list disappear and appear
+  const x = document.getElementById('menu-item');
+  if (x.style.display === 'flex') {
+    x.style.display = 'none';
+  } else {
+    x.style.display = 'flex';
+  }
+
+  const z = document.getElementById('welcome-top');
+  if (z.style.display === 'none') {
+    z.style.display = 'flex';
+  } else {
+    z.style.display = 'none';
+  }
+
+  const q = document.getElementById('open-btn');
+  if (q.style.display === 'none') {
+    q.style.display = 'flex';
+  } else {
+    q.style.display = 'none';
+  }
+
+  const k = document.getElementById('close-btn');
+  if (k.style.display === 'flex') {
+    k.style.display = 'none';
+  } else {
+    k.style.display = 'flex';
+  }
+}
+
+function myFunction2() {
+  // add onclick event for close button
+  const x = document.getElementById('menu-item');
+  if (x.style.display === 'flex') {
+    x.style.display = 'none';
+  } else {
+    x.style.display = 'flex';
+  }
+
+  const z = document.getElementById('welcome-top');
+  if (z.style.display === 'none') {
+    z.style.display = 'flex';
+  } else {
+    z.style.display = 'none';
+  }
+
+  const q = document.getElementById('open-btn');
+  if (q.style.display === 'none') {
+    q.style.display = 'flex';
+  } else {
+    q.style.display = 'none';
+  }
```

Comment:
- [ ] [_OPTIONAL_] It's best practice to name variables a meaningful name, please consider changing that.
- [ ] [_OPTIONAL_] One of the project requirements is when one of the links is clicked from the menu it should go to the appropriate part of the page which you implemented nicely but the mobile menu should disappear which you didn't, Kindly fix that.

![image](https://user-images.githubusercontent.com/59416136/167709053-bf272d9c-a39d-49e8-97bd-7c867e10ed46.png)